File: src/Resources/GenerateCSV.java
package Resources;

import java.io.File;
import java.io.FileWriter;
import java.io.IOException;

public class GenerateCSV {
    private static final int SIZE = 100;
    private static final String PATH = "test.csv";
    public static void main(String[] args) throws IOException {
        File f = new File(PATH);
        FileWriter fw = new FileWriter(f, false);
        for(int i = 0; i < SIZE; i++) {
            fw.write((Math.random() < 0.5 ? "0" : "1") + ",");
        }
        fw.close();
    }
}

File: src/Resources/GenerateCSV.py
# Generates a random CSV file of 100 1s and 0s
# Used for testing purposes

import random

if __name__ == "__main__":
    with open("test.csv", "w") as file:
        for i in range(100):
            file.write(str(random.randint(0, 1)) + ",")